### PR TITLE
Working against otel-java 0.6.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by iNikem**

### CODE REVIEW

**Comment by iNikem** [comment]
Damn! Wrong push :( @trask @jkwatson Can I still ask you for the review of this PR, please?

File: instrumentation/opentelemetry-api-beta/opentelemetry-api-beta.gradle [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apply from: "$rootDir/gradle/instrumentation.gradle"
 
 dependencies {
-  // this instrumentation needs to be able to be able to reference both the OpenTelemetry API
+  // this instrumentation needs to be able to reference both the OpenTelemetry API
```
Comment (trask) [inline instrumentation/opentelemetry-api-beta/opentelemetry-api-beta.gradle:4]: 😄

File: instrumentation/opentelemetry-api-beta/src/main/java/io/opentelemetry/auto/instrumentation/opentelemetryapi/trace/Bridging.java [hunk 2]
```diff
@@ -100,17 +101,20 @@ public static SpanContext toUnshaded(final io.opentelemetry.trace.SpanContext sh
...
-    final Map<String, io.opentelemetry.common.AttributeValue> shadedAttributes = new HashMap<>();
-    for (final Map.Entry<String, AttributeValue> entry : unshadedAttributes.entrySet()) {
-      final AttributeValue value = entry.getValue();
-      final io.opentelemetry.common.AttributeValue shadedValue = toShadedOrNull(value);
-      if (shadedValue != null) {
-        shadedAttributes.put(entry.getKey(), shadedValue);
-      }
-    }
-    return shadedAttributes;
+  public static io.opentelemetry.common.Attributes toShaded(final Attributes unshadedAttributes) {
```
Comment (trask) [inline instrumentation/opentelemetry-api-beta/src/main/java/io/opentelemetry/auto/instrumentation/opentelemetryapi/trace/Bridging.java:104]: (in the future) this probably is better in common package, like you did for LabelsShader

File: testing/src/main/groovy/io/opentelemetry/auto/test/asserts/TagsAssert.groovy [hunk 3]
```diff
@@ -18,18 +18,19 @@ package io.opentelemetry.auto.test.asserts
...
 import groovy.transform.stc.SimpleType
 import io.opentelemetry.common.AttributeValue
-
+import io.opentelemetry.common.ReadableAttributes
+import io.opentelemetry.common.ReadableKeyValuePairs
 import java.util.regex.Pattern
 
 class TagsAssert {
-  private final Map<String, AttributeValue> tags
+  private final ReadableAttributes tags
```
Comment (trask) [inline testing/src/main/groovy/io/opentelemetry/auto/test/asserts/TagsAssert.groovy:26]: oh ya i never got around to renaming `tags` to `attributes` in tests

File: testing/src/main/groovy/io/opentelemetry/auto/test/asserts/TagsAssert.groovy [hunk 4]
```diff
@@ -82,12 +83,18 @@ class TagsAssert {
...
+      @Override
+      void consume(String key, AttributeValue value) {
+        allTags.add(key)
+      }
+    })
+    Set<String> unverifiedTags = new TreeSet(allTags)
+    unverifiedTags.removeAll(assertedTags)
+    // tags and assertedTags are included to provide better context in the error message.
+    //containsAll because we may assert more than span actually has
+    assert unverifiedTags.isEmpty() && assertedTags.containsAll(allTags)
```
Comment (trask) [inline testing/src/main/groovy/io/opentelemetry/auto/test/asserts/TagsAssert.groovy:97]: staring at this way too long... 😂

how about this comment instead:

```
// the first and second condition in the assert are exactly the same
// but both are included in order to provide better context in the error message
```

**Pull request merged**